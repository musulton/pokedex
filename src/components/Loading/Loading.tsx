import {Container, Text} from "@/styles/commonStyled";

const Loading = () => {
    return (
        <Container>
            <Text>Loading...</Text>
        </Container>
    )
}

export default Loading;
