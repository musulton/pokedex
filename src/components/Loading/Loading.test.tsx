import {render} from "@testing-library/react";

import Loading from "./Loading";

it("should render Loading correctly", () => {
    const {container} = render(<Loading />)

    expect(container.firstChild).toMatchSnapshot();
})
