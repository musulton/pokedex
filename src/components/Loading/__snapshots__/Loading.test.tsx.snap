// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Loading correctly 1`] = `
<div>
  <div
    class="sc-249c5c88-0 bHOtmB"
  >
    <p
      class="sc-249c5c88-1 dUHQTv"
    >
      Loading...
    </p>
  </div>
</div>
`;
